ymcatwincities/openy
# Collapsible Paragraph Items During Node Edit

Task: https://trello.com/c/Nn2C3ghI/18-collapsible-paragraph-items-during-node-edit-score-6

*Steps to test:*
- Install new site or run database updates on the existing one.
- Edit any entity of the content types, which contain paragraphs.
- Check that all paragraphs are collapsed.
